src/views/record-browser/table-data/table-data.component.js

```diff
@@ -67,9 +68,35 @@
 		}
 
 		function getRows(columns, data) {
-
 			// Convert document objects into arrays with the same order as `columns`
 			return data.map(document => columns.map(column => document[column.name]));
 		}
+
+		// Modal that pops up when clicking on delete row icon
+		$scope.showConfirm = function(row) {
+			var result = modalHelper.confirmDelete('record');
+			var row_id = row[0];
+			result.then(function() {
```

Comment:
`result` could be named something more descriptive.  However, it makes sense to get rid of the variable altogether and write `modalHelper.confirmDelete('record').then(...)`